docs/guides/deploy/hosting-frontend.md

```diff
@@ -35,3 +35,17 @@ Make sure you pick the backend you want to use for the deploy.
 -   **Google Cloud Platform** If you would like use GCP to host the frontend you can follow this [guide](https://cloud.google.com/storage/docs/hosting-static-website). There you will create a bucket, a load balancer and a CDN.
 
 -   **Amazon Web Services** If you prefer to use AWS you can watch this [video](https://www.youtube.com/watch?v=BpFKnPae1oY&ab_channel=AmazonWebServices) it will explain how to create an S3 bucket and how to route traffic using the AWS DNS service, Route 53.
+
+## Hosting your Frontend at Vercel
+
+It is super simple. So simple even Rico can do it.
```

Comment:
Super fun! 😂

I like the flavor you have here, but this might be a tiiiny bit to internal, so I would feel good to merge this if you could keep the same feel, but get your name out of it! 😁

Reply:
Rico has bee eliminated from. 😢 

🤣